ups/sphgeom.table

```diff
@@ -1,2 +1,7 @@
+setupRequired(base)
+setupRequired(sconsUtils)
+setupOptional(doxygen)
+
 envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
 envPrepend(DYLD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
+envPrepend(PYTHONPATH, ${PRODUCT_DIR}/python)
```

Comment:
Thanks for the link. Note however that the templates package did not receive this update (maybe it's not the primary source of documentation for this sort of thing, but I didn't know where else to look).